src/components/organisms/AdminVenueView/components/Timing/SpaceTimingPanel.tsx

```diff
@@ -0,0 +1,47 @@
+import React from "react";
+
+import { AnyVenue } from "types/venues";
+
+import { WithId } from "utils/id";
+
+import { AdminPanel } from "components/organisms/AdminVenueView/components/AdminPanel";
+import { AdminShowcase } from "components/organisms/AdminVenueView/components/AdminShowcase";
+import { AdminSidebar } from "components/organisms/AdminVenueView/components/AdminSidebar";
+import {
+  AdminSidebarFooter,
+  AdminSidebarFooterProps,
+} from "components/organisms/AdminVenueView/components/AdminSidebarFooter/AdminSidebarFooter";
+import { AdminSidebarTitle } from "components/organisms/AdminVenueView/components/AdminSidebarTitle";
+import { SpaceTimingForm } from "components/organisms/SpaceTimingForm";
+
+import { LoadingPage } from "components/molecules/LoadingPage";
+
+import { EventsView } from "../EventsView";
+
+import "./SpaceTimingPanel.scss";
+
+export interface SpaceTimingPanelProps extends AdminSidebarFooterProps {
+  venue: WithId<AnyVenue>;
+}
+
+export const SpaceTimingPanel: React.FC<SpaceTimingPanelProps> = ({
+  venue,
+  ...sidebarFooterProps
+}) => {
+  if (!venue) {
+    return <LoadingPage />;
+  }
+
+  return (
+    <AdminPanel className="SpaceTimingPanel">
+      <AdminSidebar>
+        <AdminSidebarTitle>Plan your event</AdminSidebarTitle>
+        <AdminSidebarFooter {...sidebarFooterProps} />
```

Comment:
Can you please move this one line lower, below `<SpaceTimingForm venue={venue} />` just to represent the order of rendering on the actual screen.

Reply:
It usually is at the top in all the other forms just because of the actual forms longevity.
The order of rendering on the screen is the same as in the code, CSS doesn't modify it.
It just makes the first rendered component show up at the bottom of the screen, followed by the rest.